components/TenancySummary.tsx

```diff
@@ -11,40 +10,50 @@ interface Props {
     tenureType?: string;
     startDate?: string | Date;
   };
+  extraRows?: { key: string; value: React.ReactNode }[];
 }
 
 export const TenancySummary: React.FunctionComponent<Props> = (props) => {
-  const { details } = props;
+  const { details, extraRows } = props;
   const { address, tenants, tenureType, startDate } = details || {};
 
   const loading = "Loading...";
 
+  const rows = [
+    {
+      key: "Address",
+      value: address ? address.join(", ") : loading,
+    },
+    {
+      key: "Tenants",
+      value: tenants ? tenants.join(", ") : loading,
+    },
+    {
+      key: "Tenure type",
+      value: tenureType ? tenureType : loading,
+    },
+    {
+      key: "Tenancy start date",
+      value: startDate
+        ? typeof startDate === "string"
+          ? startDate
+          : formatDate(startDate, "d MMMM yyyy")
+        : loading,
+    },
+  ];
+
   return (
     <>
       <SummaryList
         className="govuk-summary-list--no-border mat-tenancy-summary"
-        rows={[
-          {
-            key: "Address",
-            value: address ? address.join(", ") : loading,
-          },
-          {
-            key: "Tenants",
-            value: tenants ? tenants.join(", ") : loading,
-          },
-          {
-            key: "Tenure type",
-            value: tenureType ? tenureType : loading,
-          },
-          {
-            key: "Tenancy start date",
-            value: startDate
-              ? typeof startDate === "string"
-                ? startDate
-                : formatDate(startDate, "d MMMM yyyy")
-              : loading,
-          },
-        ]}
+        rows={
+          extraRows
+            ? [
+                ...rows,
+                ...((extraRows as unknown) as { key: string; value: string }[]),
+              ]
+            : rows
```

Comment:
Should be able to write this like `[...rows, ...(((extraRows || []) as unknown) as { key: string; value: string }[])]`, to be flatter.